Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                <locus from="55va" to="79vb"/>
+                                <title type="complete" ref="LIT1146Argano#Wednesday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="80ra" to="86rb"/>
+                                <locus from="87ra" to="104vb"/>
+                                <title type="complete" ref="LIT1146Argano#Thursday"></title>
+                                <note>The text on <locus target="#86r"/> is written by a different hand.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.5.">
+                                <locus from="105ra" to="127va"/>
+                                <title type="complete" ref="LIT1146Argano#Friday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.6">
+                                <locus from="127vb" to="139vb"/>
+                                <title type="complete" ref="LIT1146Argano#Saturday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.7">
+                                <locus from="139vb" to="151rb"/>
+                                <title type="complete" ref="LIT1146Argano#Sunday"></title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39</citedRange></bibl>
+                                    notes that for the original redactor of the text version attested in this manuscript,
+                                    the reading for Sunday began only on <locus target="143ra"/>.
+                                    (This is clear from the presence of <foreign xml:lang="gez">ḥarag</foreign> only there.)
+                                </note>
+                            </msItem>
+                        </msItem>
+                      
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">153</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>235</height>
+                                        <width>20</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <signatures>The quires have Ethiopic quire numbers on the recto of the first leaf and the
```

Comment:
It would be interesting to have more about the position of the quire numbers: upper corner left / right..?